[Bug]: 0.48.0 breaks button style imports

### Preflight Checklist

- [X] I have read the [Contributing Guidelines](https://github.com/lyne-design-system/lyne-components/blob/main/CONTRIBUTING.md) for this project.
- [X] I agree to follow the [Code of Conduct](https://github.com/lyne-design-system/lyne-components/blob/main/CODE_OF_CONDUCT.md) that this project adheres to.
- [X] I have searched the [issue tracker](https://github.com/orgs/lyne-design-system/projects/4) for an issue that matches the one I want to file, without success.

### Bug type

Functionality

### It affects the following packages

Components

### Which version of Lyne Components are you using

0.48.0

### Operating system

macOS

### Browser / Browser version

Chromium

### Input mode

None

### Additional settings

_No response_

### What happened?

Since 0.48.0, and probably PR #2475, our <sbb-button>, respectively its style imports break with the following error:
_node_modules/@sbb-esta/lyne-components/button/common/index.d.ts:2:46 - error TS2307: Cannot find module './button-common.scss?lit&inline' or its corresponding type declarations._
See the stackblitz example linked below.

I'm not sure if this is an "Angular only" issue or not - so far I don't have a deep enough understanding of how Lit components work. 😉
Thanks in advance!

### Steps to reproduce the issue

1. Set up a basic Angular project
2. Install Lyne
3. Try to use a button (any variant will do)
4. See the error in the console

Minimal stackblitz:
https://stackblitz.com/edit/angular-at-nxjrka?file=src%2Fmain.ts

### Relevant log output

_No response_

--- a/src/components/button/common/button-common.ts
+++ b/src/components/button/common/button-common.ts
@@ -16,6 +16,12 @@ import {
 
 import '../../icon';
 
+export { default as buttonCommonStyle } from './button-common.scss?lit&inline';
+export { default as buttonPrimaryStyle } from './primary-button.scss?lit&inline';
+export { default as buttonSecondaryStyle } from './secondary-button.scss?lit&inline';
+export { default as buttonTertiaryStyle } from './tertiary-button.scss?lit&inline';
+export { default as buttonTransparentStyle } from './transparent-button.scss?lit&inline';
+
 export type SbbButtonCommonElement = SbbButtonCommonElementMixinType & SbbActionBaseElement;
 
 export type SbbButtonSize = 'l' | 'm';

--- a/src/components/button/common/index.ts
+++ b/src/components/button/common/index.ts
@@ -1,7 +1,1 @@
 export * from './button-common';
-
-export { default as buttonCommonStyle } from './button-common.scss?lit&inline';
-export { default as buttonPrimaryStyle } from './primary-button.scss?lit&inline';
-export { default as buttonSecondaryStyle } from './secondary-button.scss?lit&inline';
-export { default as buttonTertiaryStyle } from './tertiary-button.scss?lit&inline';
-export { default as buttonTransparentStyle } from './transparent-button.scss?lit&inline';

